spyder/plugins/completion/providers/languageserver/conftabs/linting.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, parent):
             'flake8/filename',
             alignment=Qt.Horizontal,
             word_wrap=False,
-            placeholder=_("Check test files: test_.*\.py"),
+            placeholder=_("Check test files: test_.*\\.py"),
```

Comment:
I understand that this gives a `SyntaxWarning` in Python, but is it really necessary to add the extra `\` for this Flake8 option?

If not, then perhaps we should change the example to something else.